examples/05_Collections/02_Set.md

```diff
@@ -3,9 +3,9 @@
 A [set](https://kotlinlang.org/docs/reference/collections.html) is an unordered collection that does not support duplicates. For creating sets, there are functions `setOf()` and `mutableSetOf()`. A read-only view of a mutable set can be obtained by casting it to `Set`.  
 
 ```run-kotlin
-val openIssues: MutableSet<String> = mutableSetOf("uniqueDescr1", "uniqueDescr2", "uniqueDescr3") // 1
+val openIssues: MutableSet<String> = mutableSetOf("uniqueDescr1", "uniqueDescr2", "uniqueDescr3") 
 
-fun addIssue(uniqueDesc: String): Boolean {                                                       
+fun addIssue(uniqueDesc: String): Boolean {                                                       // 1
```

Comment:
Thanks for pointing this out. However, with these changes we're having two descriptions for `addIssues()` and no description from `mutableSetOf()`.
I'd suggest leaving the  `// 1` comment where it is now and change the description to
**1. Creates a `Set` with given elements.**